src/support/assertRender.ts

```diff
@@ -140,6 +142,24 @@ export default function assertRender(actual: DNode | DNode[], expected: DNode |
 				throwAssertionError(actual.widgetConstructor, expected.widgetConstructor, `WNodes do not share constructor`, message);
 			}
 		}
+		/* Inject a custom comparator for class names */
+		const expectedClasses: SupportedClassName[] | undefined = expected.properties && (expected.properties as any).classes;
+		if (expectedClasses && !isCustomDiff(expectedClasses)) {
+			(expected.properties as any).classes = compareProperty((value?: SupportedClassName[]) => {
```

Comment:
wouldn't it be more efficient to have a single function outside the scope of the function, so you only get one function instead of a new function scope every invocation?

Reply:
The comparator doesn't have access to the expected classes when it runs, so I think I need to create a function here to capture that. I may be missing something obvious but I don't see how I could do that without creating a function, unless this change was added to the comparison logic itself.